Extension/src/Debugger/configurationProvider.ts

```diff
@@ -24,7 +24,7 @@ nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFo
 const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
 function isDebugLaunchStr(str: string): boolean {
-    return str === "(gdb) Launch" || str === "(lldb) Launch" || str === "(Windows) Launch";
+    return str.startsWith("(gdb) ") || str.startsWith("(lldb) ") || str.startsWith("(Windows) ");
```

Comment:
I don't know if this is the right fix. You could end up with the Attach script. you need to also check that the `request` field is `launch`

Reply:
I couldn't get a repro where "Attach" was hit by this code. Do you know a repro or is it in case the isDebugLaunchStr is used incorrectly later on in a different case?

Reply:
I added code to check the request field...although I still don't know how to hit a case where the request is not "launch".

Reply:
I thought about this some more. Is your goal to get the first `launch` configuration? I think you can do this without doing string comparisons with the title.

You should be able to use `type` as `cppdbg` (or `cppvsdbg` for VC++) and the `request` field and make sure that `pipeTransport` object is null. 

I don't know why Griffin originally chose the name, probably because it was easier for his case.

Reply:
That doesn't work. Our build and debug tasks have the same type, etc.